app.js

```diff
@@ -10,86 +11,116 @@ const result = document.querySelector(".result");
 const priceShoes = document.querySelector(".price");
 const carosel = document.querySelectorAll(".carosel img");
 
-const addToCartButton = document.querySelector('.add_toCart');
-const productInBasket = document.querySelector('.product_added_hide');
-const emptyBasketText = document.querySelector('.cart_empty_text');
+const addToCartButton = document.querySelector(".add_toCart");
```

Comment:
you can move all of the querySelectors to a separate file, called `css-selectors.js` and include that file in the html